frontends/web/src/globals.d.ts

```diff
@@ -17,6 +17,8 @@
 export declare global {
     interface Window {
         qt?: unknown;
-        android?: unknown;
+        android?: (query: string) => void;
+        onAndroidCallResponse?: unknown;
```

Comment:
can be `(queryID: number, response: string) => void` instead of `unknown`, right?